doc/orchestrator.rst

```diff
@@ -1,60 +1,324 @@
 ************
 Orchestrator
 ************
-
-
-The ``Orchestrator`` is an in-memory database that is launched prior to all other
-entities within an ``Experiment``. The ``Orchestrator`` can be used to store and retrieve
-data during the course of an experiment and across multiple entities. In order to
-stream data into or receive data from the ``Orchestrator``, one of the SmartSim clients
-(SmartRedis) has to be used within a Model.
+========
+Overview
+========
+The orchestrator is an in-memory database with features built for
+AI-enabled workflows including online training, low-latency inference, cross-application data
+exchange, online interactive visualization, online data analysis, computational
+steering, and more. The ``Orchestrator`` can be thought of as a general
+feature store capable of storing numerical data, ML models, and scripts.
+The orchestrator is capable of performing inference and script evaluation using data in the feature store.
+Any SmartSim ``Model`` or ``Ensemble`` model can connect to the
+``Orchestrator`` via the :ref:`SmartRedis<smartredis-api>`
+client library to transmit data, execute ML models, and execute scripts.
+
+SmartSim offers two types of orchestrator deployments: :ref:`clustered deployment<Clustered Deployment>` and
+:ref:`colocated deployment<Colocated Deployment>`. Continue to the respective deployment sections for
+information on deployment types.
+
+Below is a diagram demonstrating an orchestrator as a general feature store
+capable of storing numerical data (tensors and datasets), AI models, and scripts (TorchScript).
+Combined with the SmartRedis clients, the ``Orchestrator`` is capable of hosting and executing
+AI models written in Python on CPU or GPU. The ``Orchestrator`` supports models written with
```

Comment:
It feels like we're repeating ourselves from the first overview paragraph by restating the capabilities. Consider reducing amount of restated capabilities.

Perhaps, shifting diagram up before the line `Any SmartSim ``Model`` or ``Ensemble`` model can connect to the...` would lead cleanly into the diagram and avoid that need to repeat?